crawler-cli/src/main/java/org/gbif/crawler/pipelines/dwca/DwCAToAvroCommandVerification.java

```diff
@@ -0,0 +1,153 @@
+package org.gbif.crawler.pipelines.dwca;
+
+import org.gbif.common.messaging.api.messages.DwcaValidationFinishedMessage;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.URI;
+import java.nio.file.Files;
+import java.util.Objects;
+
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.fs.FileSystem;
+import org.apache.hadoop.fs.Path;
+import org.eclipse.jdt.internal.core.Assert;
+
+/**
+ * Verifies the configuration parameters like existence of input and output path, and enables with calculated configuration parameters helping DwCAToAvro Conversion
+ */
+public class DwCAToAvroCommandVerification {
```

Comment:
I think we need to keep clear the intention of this class, we shouldn't be doing on each message:
 Objects.requireNonNull(configuration, "Configuration cannot be null");
maybe we just need to create/check target directories for the dataset being crawled and forget about the rest since it can slow down the process

Reply:
removed the not null checks.